napari/layers/shapes/_tests/test_shape_list.py

```diff
@@ -23,6 +23,27 @@ def test_adding_to_shape_list():
     assert shape_list.shapes[0] == shape
 
 
+def test_shape_list_outline():
+    """Test ShapeList outline method."""
+    np.random.seed(0)
+    data = 20 * np.random.random((4, 2))
+    shape = Rectangle(data)
+    shape_list = ShapeList()
+
+    shape_list.add(shape)
+
+    outline_by_index = shape_list.outline(0)
+    assert isinstance(outline_by_index, tuple)
+
+    outline_by_index_list = shape_list.outline(np.array(0))
```

Comment:
To test both branches, does it make sense to also add something of type Sequence? like `[0]`